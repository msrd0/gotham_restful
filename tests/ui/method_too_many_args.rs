#[macro_use] extern crate gotham_restful;

#[derive(Resource)]
#[resource(read_all)]
struct FooResource;

#[read_all(FooResource)]
fn read_all(_id : u64)
{
}

fn main()
{
}
