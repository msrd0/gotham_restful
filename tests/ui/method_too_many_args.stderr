error: Too many arguments
 --> $DIR/method_too_many_args.rs:8:13
  |
8 | fn read_all(_id : u64)
  |             ^^^
